lib/krb5/pac.c

```diff
@@ -911,6 +932,21 @@ krb5_pac_verify(krb5_context context,
 			      privsvr);
 	if (ret)
 	    return ret;
+
+	if (pac->ticket_sign_data.length != 0) {
```

Comment:
Remind me, I'm not across the latest protocol revisions, but we're going to require a ticket checksum always, not just when S4U2Proxy is present, because this is the moral replacement for KRB5SignedPath?

Reply:
Well, what's always? When else does a KDC get to look at a service ticket except S4U2Proxy and renewal/validation? We could make other uses of that, but I'm not aware of existing use case.

Reply:
user-to-user?

Reply:
(sorry, that is a TGT, I just still get confused by this feature)